src/main/kotlin/tech/relaycorp/poweb/PoWebClient.kt

```diff
@@ -29,7 +29,6 @@ import kotlinx.coroutines.delay
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.FlowCollector
 import kotlinx.coroutines.flow.flow
-import mu.KotlinLogging
```

Comment:
Can you please remove the dependency from Gradle? I guess this was the only place that used it.